src/restLevelClient/generateRestLevel.ts

```diff
@@ -28,9 +30,20 @@ export async function generateRestLevelClient() {
     }
   });
 
+  model.operationGroups.some(og =>
+    og.operations.some(
+      o => o.extensions && o.extensions["x-ms-long-running-operation"]
+    )
+  );
```

Comment:
Is this duplicate with what's in hasPollingOperations.ts ?

Reply:
Yup, its a left over. Removing